mock/py/mockbuild/package_manager.py

```diff
@@ -156,7 +157,22 @@ def update(self, *args, **kwargs):
     # pylint: disable=unused-argument
     @traceLog()
     def builddep(self, *args, **kwargs):
-        return self.execute('builddep', returnOutput=1, *args)
+        try:
+            result = self.execute('builddep', returnOutput=1, *args)
+        except (FileNotFoundError) as error:
+            er = str(error)
+            if "builddep" in er:
+                print(error)
+                print("""
+Error:      dnf-utils is not installed. Dnf-utils is needed to complete this action.
+            To install dnf-utils use one of the commands below:
+            $ dnf install dnf-utils
+            or
+            $ yum install dnf-utils""")
```

Comment:
Shouldn't here be `yum install yum-utils`?

Reply:
I have to admit I don't know for sure. Both ```yum install dnf-utils``` and ```yum install yum-utils``` seems to work. But I guess it should be the later.
I'll create a fix.

Reply:
It IMO depends how mock behaves on rhel-7;  if `/usr/bin/yum-builddep` is used or not.  I don't think it is OK to rely on `dnf-utils` in rhel-7.

Reply:
That said, I'm not 100% sure whether the `or yum install ...` is even needed :-)